package.json

```diff
@@ -67,7 +68,7 @@
     "@uniswap/redux-multicall": "^1.1.8",
     "@uniswap/router-sdk": "^1.3.0",
     "@uniswap/sdk-core": "^3.2.0",
-    "@uniswap/smart-order-router": "^3.10.0",
+    "@uniswap/smart-order-router": "^3.9.0",
```

Comment:
did you mean to revert this?

Reply:
i plan to revert this before merging, this is just to prove that it's working